src/main/java/com/epam/ta/reportportal/dao/LogTypeRepository.java

```diff
@@ -18,8 +18,19 @@
 
 import com.epam.ta.reportportal.entity.log.ProjectLogType;
 import java.util.List;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface LogTypeRepository extends ReportPortalRepository<ProjectLogType, Long> {
 
   List<ProjectLogType> findByProjectId(Long projectId);
+
+  @Query("SELECT COUNT(log.id) > 0 FROM ProjectLogType log " +
```

Comment:
let's use new java
""" 
  SELECT ....
"""
format for queries

Reply:
done